source/_components/google_assistant.markdown

```diff
@@ -30,6 +30,8 @@ To use Google Assistant, your Home Assistant configuration has to be externally
 Since release 0.80, the `Authorization Code` type of `OAuth` account linking is supported. To migrate your configuration, you need:
 
 1. Change your `Account linking` setting in [Actions on Google console](https://console.actions.google.com/), look for the `Advanced Options` in the bottom left of the sidebar.
+    - Under `Create an Action`, under the build section, modify `Add fulfillment URL` to `https://[YOUR HOME ASSISTANT         URL:PORT]/api/google_assistant`, where `[YOUR HOME ASSISTANT URL:PORT]` is the domain / IP address and the port under which your Home Assistant instance is reachable.
+    
```

Comment:
You are breaking up the list by adding a blank line here.

Reply:
Sorry, fixed that :)